etcdctl/README.md

```diff
@@ -1083,6 +1083,47 @@ echo ${transferee_id}
 # Leadership transferred from 45ddc0e800e20b93 to c89feb932daef420
 ```
 
+### DOWNGRADE \<subcommand\>
+
+Downgrade provides commands to downgrade cluster version
```

Comment:
nit: "." at the end of this (and following sentences".

Reply:
Done